app/frontend/styles/_application-card.scss

```diff
@@ -84,4 +84,16 @@
       margin-top: auto;
     }
   }
+
+  &__course_type {
+    margin-bottom: 10px;
+
+    font-weight: 700;
```

Comment:
we should use the govuk styles rather than define these with specific values unless it's absolutely necessary. something like:

`font-weight: $govuk-font-weight-bold;`

Reply:
Done